client/src/pages/BusinessProfile.js

```diff
@@ -4,13 +4,13 @@ import React, { useState, useContext, useEffect } from 'react'
 import { useHistory } from 'react-router-dom'
 // Custom Imports
 import ContentHeader from '../components/ContentHeader'
-import { AuthContext } from '../contexts/GlobalContext'
+import { AppContext } from '../contexts/AppContext'
 
 const BusinessProfile = () => {
   const history = useHistory()
 
   // Import User State Object from Context
-  const { userState, setUserState } = useContext(AuthContext)
+  const { user, setUser } = useContext(AppContext)
```

Comment:
Hi @cynthiaeddy, the `userId` is not accessible from context during the `saveNewBusiness` function and is not saved as a new document in the businesses collection. See images with a newly added user, Ben Walker with Walker Cutz, as compared to previous user Billy Stone with Stone Cutz below.

![image](https://user-images.githubusercontent.com/32943879/141696737-5103e035-cad8-42d0-b081-d7b852d8992e.png)